mkdocs.yml

```diff
@@ -18,6 +18,7 @@ theme:
   name: material
   icon:
     repo: material/github
+    annotation: material/help-circle
```

Comment:
Just wondering, what do the added extensions do ?

Reply:
It allows adding small annotations like that to reduce line noise : 

![image](https://github.com/user-attachments/assets/b9b3f03c-6555-41da-8b2d-2f3c695474f9)

Reply:
Great ! Thanks for the explanations ! 👍